docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -172,7 +181,71 @@ will be considered by default. Choose _All Files (\*.*)_ from the drop down, if
 image:images/sql/odbc/dsn_editor_security_cert.png[]
 
 [float]
-===== 2.4 Logging parameters
+===== 2.4 Connection parameters
+The connection configuration can further be tweaked by the following parameters, but these should only need to be changed in rare cases.
+
+* Request timeout (s)
++
+The maximum number of seconds a request to the server can take. The value 0 disables disables the timeout.
```

Comment:
Double `disables`.

Reply:
Maybe: `The maximum duration in seconds for a request to the server.`

Reply:
Fixed the doubling and adopted the reformulation.